core/data/stories.md

```diff
@@ -1037,4 +1037,83 @@
   - utter_ask_assess_to_answer
 * done
   - utter_please_visit_again
-  - action_qa_goodbye
\ No newline at end of file
+  - action_qa_goodbye
+
+## Test navigation - do nothing
+* navigate_test_locations
+  - action_test_navigation_explanations
+  - utter_ask_test_navigation__continue_no_assessment
+* later
+  - utter_test_navigation__come_back
+  - action_goodbye
+
+## Test navigation - ask question - failure
+* navigate_test_locations
+  - action_test_navigation_explanations
+  - utter_ask_test_navigation__continue_no_assessment
+* ask_question
+  - question_answering_form
+  - form{"name": "question_answering_form"}
+  - form{"name": null}
+  - slot{"question_answering_status": "failure"}
+  - utter_question_answering_error
+  - utter_ask_assess_after_error
```

Comment:
Can you check if these stories impact the QA stories which have the same featurized slots?

Reply:
I can check, but basically I copied them here so what follows is the same. If that throws off the TED policy, I loose all faith I had in it...

Reply:
Nvm, seems fine